### DISCUSSION TITLE
Change ASSA Script Resolution

### DISCUSSION THREAD

**Discussion by acteiaaaaaaaaaaaaaaaaaaa** [SubtitleEdit/subtitleedit#8630] [General]
![image](https://github.com/user-attachments/assets/98aeac10-5b78-4a18-bcd2-dbd36d8e8e01)
Please return this function. It is only available in version 4.0.5 and is very useful.

**Comment by niksedk** [comment]
Should still be there - right-click in the list view, ASSA tools

![image](https://github.com/user-attachments/assets/650b59e4-7524-48e1-a309-ee795b21d676)
